modules/default/calendar/README.md

```diff
@@ -53,25 +53,36 @@ The `colored` property gives the option for an individual color for each calenda
 #### Default value:
 ````javascript
 config: {
-    	colored: false,
+    colored: false,
 	calendars: [
 		{
 			url: 'http://www.calendarlabs.com/templates/ical/US-Holidays.ics',
 			symbol: 'calendar',
+			auth: {
```

Comment:
I just discovered, that this example is at the wrong place...
I'll take care of this tomorrow.